pylint/test/functional/disable_wrong_import_order.py

```diff
@@ -0,0 +1,11 @@
+"""Checks that disabling 'wrong-import-order' on an import prevents subsequent
+imports from being considered out-of-order in respect to it but does not prevent
+it from being considered for 'ungrouped-imports'."""
+# pylint: disable=unused-import,import-error,no-name-in-module
+
+from first_party.foo import bar # pylint: disable=wrong-import-order
+import logging
+import os.path
+import sys
+from astroid import are_exclusive
+import first_party  # [ungrouped-imports]
```

Comment:
Yeah it seems to tell the tester that the specified error is expected on the commented line. The actual message output appears in the associated `.txt` file. It seems maybe a bit redundant at first glance.